### DISCUSSION TITLE
Read only mode

### DISCUSSION THREAD

**Discussion by Darkbluetest** [Vinzent03/obsidian-git#389] [Q&A]
Hi,

I am looking for a way to share the vault to general public, so I was interested in this plugin and using a Git to upload the vault. But I would need that the reader will not need to make a repository in their devices neither to be able to write (makes pushs). Is it possible or could it be done, a read only mode which only allow to make pulls?

Thanks

**Comment by Vinzent03** [comment]
That's not part of this plugin. This only interacts with git. I think what you describe is a default public GitHub repo. Everybody can read, but only you can write.

**Comment by Darkbluetest** [comment]
Yes, but I am not sure Obsidian can read the GitHub repository directly, without being the owner (and thus using Obsidian Git)

**Reply by Vinzent03** [reply]
It uses normal git, so it can. You have to initially clone it and pull as always afterwards.
